riva/client/argparse_utils.py

```diff
@@ -85,6 +85,12 @@ def add_asr_config_argparse_parameters(
         type=float,
         help="Threshold value for likelihood of blanks before detecting end of utterance",
     )
+    parser.add_argument(
+        "--custom-configuration",
+        default="",
+        type=str,
+        help="Add custom configurations to be sent to the custom backends. <key:value,key:value>",
```

Comment:
similar comment here for clarity, <key:value,key:value,...>

Reply:
Updated